lib/main.dart

```diff
@@ -0,0 +1,35 @@
+import 'classes/atleta.dart';
+import 'classes/tecnico.dart';
+import 'classes/diretor.dart';
+import 'classes/time.dart';
+
+void main() {
+  Atleta ok = Atleta("Pedro", 18, 76, "121.692.176-33", "pedro", "Futsal");
```

Comment:
' ou "

escolha um e use sempre o mesmo
cara hora um não dá...

eu particularmente to gostando mais de usar só ', mas é seu critério